node/id.go

```diff
@@ -1,35 +0,0 @@
-package node
```

Comment:
what was this file used for? Never seen it lol

Reply:
nothing! it was dead code, but because it was public the linter didn't catch it.

Reply:
Wow, our API is soooo public.